python-sdk/src/astro/dataframes/load_options.py

```diff
@@ -8,56 +8,30 @@
 
 @define
 class PandasLoadOptions(LoadOptions):
-    """Pandas load options while reading and loading file"""
-
-    pass
-
-
-@define
-class PandasCsvLoadOptions(PandasLoadOptions):
     """
     Pandas load options while reading and loading csv file.
 
-    :param delimiter: Delimiter to use. Defaults to None
-    :param dtype: Data type for data or columns. E.g. ``{"a": np.float64, "b": np.int32, "c": "Int64"}`` Use str or
+    :param delimiter: valid file type[CSV] - Delimiter to use. Defaults to None
```

Comment:
I think we need to have a way to pass all other supported parameters for Pandas as `kwargs` here. We should add this change as part of this PR.

Reply:
resolved.